system/Commands/Utilities/Routes/AutoRouterImproved/AutoRouteCollector.php

```diff
@@ -122,6 +122,7 @@ private function addFilters($routes)
             $filtersShortest = $filterCollector->get($route['method'], $routePath . $sampleUri);
 
             // Get common array elements
+            $filters           = [];
             $filters['before'] = array_intersect($filtersLongest['before'], $filtersShortest['before']);
             $filters['after']  = array_intersect($filtersLongest['after'], $filtersShortest['after']);
```

Comment:
Late to comment this, I prefer this kind of style.
```php
$filters = [
          'before' => array_intersect($filtersLongest['before'], $filtersShortest['before']),
          'after' => array_intersect($filtersLongest['after'], $filtersShortest['after']),
];
```

Reply:
Yeah, I realised later on that I can do it that way. You can send a PR to clean that up.